atomium/templates/item_list/item_list.component.inc

```diff
@@ -128,6 +133,17 @@ function atomium_preprocess_item_list(array &$variables, $hook) {
       }
     }
 
+    if (isset($item['children'])) {
+      $children = $item['children'];
+      unset($item['children']);
+      $item['children'] = array(
+        '#theme' => 'item_list',
+        '#items' => $children,
```

Comment:
```suggestion
        '#items' => $item['children'],
```

Reply:
The line needs to swap the order been the main item and the children items, you can test the result with and without you will see!